tests/pipeline/test_international_markets.py

```diff
@@ -402,6 +402,16 @@ def test_explicit_specialization_matches_implicit(self):
             sessions[-1],
         )
 
+    def test_cannot_convert_volume_data(self):
+        with self.assertRaises(TypeError) as exc:
+            Pipeline({'volume': EquityPricing.volume.fx('EUR').latest})
+
+        assert_equal(
+            str(exc.exception),
+            'The .fx method cannot be called on EquityPricing.volume because '
```

Comment:
Elsewhere in the API for errors like this we write `.fx()` rather than `.fx`. See, for instance, [float64_only](https://github.com/quantopian/zipline/blob/master/zipline/pipeline/factors/factor.py#L368-L374)